modules/h12mediaBidAdapter.js

```diff
@@ -16,21 +15,30 @@ export const spec = {
   },
 
   buildRequests: function(validBidRequests, bidderRequest) {
-    const requestUrl = validBidRequests[0].params.endpointdom || DEFAULT_URL;
-    const isiframe = !((window.self === window.top) || window.frameElement);
+    const isiframe = inIframe();
     const screenSize = getClientDimensions();
     const docSize = getDocumentDimensions();
 
-    const bidrequests = validBidRequests.map((bidRequest) => {
+    return validBidRequests.map((bidRequest) => {
       const bidderParams = bidRequest.params;
-      const adUnitElement = document.getElementById(bidRequest.adUnitCode);
+      const requestUrl = bidderParams.endpointdom || DEFAULT_URL;
+      let pubsubid = bidderParams.pubsubid || '';
+      if (pubsubid && pubsubid.length > 32) {
+        utils.logError('Bidder param \'pubsubid\' should be less than 32 chars.');
```

Comment:
judging by your condition, it can also be `equal` to 32 chars.

Reply:
Fixed, the log message is changed